requirements.txt

```diff
@@ -1,20 +1,19 @@
-# We are currently targeting the versions shipped in Ubuntu 20.04.
-
-tornado>=4.5,<4.6  # http://www.tornadoweb.org/en/stable/releases.html
-psycopg2>=2.8,<2.9  # http://initd.org/psycopg/articles/tag/release/
+# Fix tornado version to be sure that it will work with our patch
+tornado==4.5.3  # http://www.tornadoweb.org/en/stable/releases.html
+psycopg2==2.9.7  # http://initd.org/psycopg/articles/tag/release/
 sqlalchemy>=1.3,<1.4  # http://docs.sqlalchemy.org/en/latest/changelog/index.html
 netifaces>=0.10,<0.11  # https://bitbucket.org/al45tair/netifaces/src/
-pycryptodomex>=3.6,<3.7  # https://github.com/Legrandin/pycryptodome/blob/master/Changelog.rst
+pycryptodomex==3.19.0  # https://github.com/Legrandin/pycryptodome/blob/master/Changelog.rst
 psutil>=5.5,<5.6  # https://github.com/giampaolo/psutil/blob/master/HISTORY.rst
-requests>=2.22,<2.23  # https://pypi.python.org/pypi/requests
-gevent==20.12.0  # http://www.gevent.org/changelog.html
-# Limit greenlet version for binary compatibility with gevent 20.12 wheels
-greenlet==1.0.0
-werkzeug>=0.16,<0.17  # https://github.com/pallets/werkzeug/blob/master/CHANGES
+requests==2.32.3  # https://pypi.python.org/pypi/requests
+gevent==23.9.0.post1  # http://www.gevent.org/changelog.html
+werkzeug<1.0
+backports.ssl-match-hostname==3.7.0.1
```

Comment:
What is this? 🤔

Reply:
Let's try if it works without it...

Reply:
Tornado appears to import it but not depend on it (?)